test/androidTest/java/org/thoughtcrime/securesms/mms/VideoCompressionTest.java

```diff
@@ -0,0 +1,61 @@
+package org.thoughtcrime.securesms.mms;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.LargeTest;
+
+import com.iceteck.silicompressorr.SiliCompressor;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.thoughtcrime.securesms.attachments.Attachment;
+import org.thoughtcrime.securesms.util.MediaUtil;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URISyntaxException;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class VideoCompressionTest {
+    @Test(expected= IOException.class)
+    public void compressVideoTest() throws Exception{
```

Comment:
Why exactly are we throwing an exception? Can we just try catch with a fail if an exception is caught?